src/components/DatePicker.js

```diff
@@ -185,7 +185,7 @@ class DatePicker extends React.Component {
     return (
       <div
         onBlur={this._handleBlur.bind(this)}
-        style={[styles.component, { fontSize: this.props.fontSize }, styles.clearFix]}
+        style={[styles.component, styles.clearFix]}
```

Comment:
We need to add `this.props.style` to this array so if a user passes in custom styles, they are applied.